updateTokenState.ts

```diff
@@ -0,0 +1,133 @@
+import './App.css';
+import {useEffect, useState} from "react";
+import
+interface SlotProps {
+    column: number
+    row: number
+    color: string
+}
+
+export let tokenState: string[][] = [];
+
+export function initTokenState() {
+    for (let r = 0; r < 6; r++) {
+        tokenState[r] = [];
+        for (let c = 0; c <7; c++) {
+            // Indicates that every slot is blue
+            tokenState[r][c] = "B";
+        }
+    }
+}
+export function isWinner({row, column, color}: SlotProps)  {
+    let rCount= 0;
+    let yCount= 0;
+
+    // Iterating through the columns and counting the number of each token
+    if(color!=="Empty")
+    {
+        for (let r = 0; r < 6; r++) {
+            if (tokenState[r][column] === "Red") {
```

Comment:
I think you only need to check for the color that was just placed, not both Red and Yellow right?